features/step_definitions/additional_cli_steps.rb

```diff
@@ -98,7 +98,7 @@
 end
 
 Given /^I have a brand new project with no files$/ do
-  in_current_dir do
+  in_current_directory do
```

Comment:
This changes are part of the first commit. I have no idea why, but I had failures locally that could be only silenced by renaming these methods. The build is failing only because of these methods that have been renamed. Give me a second, I'll delete the commit and push the branch up again.